frontend/src/metabase-lib/filter.ts

```diff
@@ -493,23 +521,20 @@ export function filterParts(
   );
 }
 
-export function isCustomFilter(
+export function isColumnFilter(
```

Comment:
nit: maybe one day we'll add first-class support for multi-column filters, and then this name might get confusing. WDYT about a name like `isStandardFilter` or something? Or maybe we can keep `isCustomFilter`

Reply:
Fixed, changed to `isStandardFilter`